weather-app/src/components/WeekForecast.js

```diff
@@ -37,19 +37,19 @@ class WeekForecast extends Component {
     const { forecast } = this.props;
 
     if (forecast.data) {
-      this.host.classList.add('week-forecast');
+      if (!this.host.classList.contains('week-forecast')) {
+        this.host.classList.add('week-forecast');
```

Comment:
I'm sorry, I was exact. I think you can avoid this check either and move adding of class to the constructor.